tools/please_pex/BUILD

```diff
@@ -1,7 +1,69 @@
-# TODO(jpoole): This should be a remove file on the plugin repo's release page and we should remove //_please:please_pex
-filegroup(
+subinclude("//build_defs:python")
+
+go_binary(
+    name = "pex_main",
+    srcs = ["pex_main.go"],
+    deps = [
+        "//third_party/go:cli-init",
+        "//third_party/go:go-logging.v1",
+        "//tools/please_pex/pex",
+    ],
+)
+
+# Combine all the Python dependencies into one zipfile and concatenate them
```

Comment:
Is this stuff still relevant? Thought maybe we were just going to make them normal dependencies?